docs/troubleshooting/node-syncing.md

```diff
@@ -70,26 +72,17 @@ If you would like to speed up connecting to other nodes and syncing, add one of
   </TabItem>
 </Tabs>
 
-These can be added in the GUI via the button, or via the CLI with `chia show -a PEER_ADDRESS:PORT` where `PORT` will usually be 8444.
-
-# Detailed explanation
-
-A regular pc can communicate **out** with endless ports-- if the user is sending a signal out -- pc opens a port -- signal goes out, pc closes the port.
-Chia uses port 8444 as instant verified communication. So an open port can allow instant communication and start the blockchain sync. Signal comes in on port 8444- that Chia pc is verified, then **both** user's pc, opens their own "communication ports ex port 8421" and that new user can now sync and now they are linked together forming part of Chia mesh.
-
-If the users port 8444 is closed, the users pc has to start sending multiple signals out and hope that a pc with open port 8444 will link with them, then the sync starts. (1) pc can only link up a few pc and with so many other Chia users coming on board, they all have to wait. Keep in mind, Chia is built on a mesh network, the blockchain is shared among all the users, not from central pc.
-
-## Dealing With Carrier-Grade NAT
```

Comment:
Can we also add this header back for the reasons listed above?

Reply:
Hey @ChuckWhitson , thank you for adding back the  "Dealing With Carrier-Grade NAT" header. 
     If we can just add back the detailed explanation I will then be able to run the ci and get this approved/merged. 
     
     We can readdress the content in the "Detailed Explanation" section to make it more understandable in the future but we would not want to remove that header at this time as that can lead to broken links.